packages/eslint-plugin/src/tests/rule.test.ts

```diff
@@ -0,0 +1,241 @@
+        },
+        {
+            code: `
+            import { action } from '@reatom/framework'
+            const doSome = action(() => {}, "doSome");
+            `
+        },
+        {
+            code: `const doSome = action();`,
+        },
+        {
+            code: `const doSome = action("do");`,
+        },
+        {
+            code: `const doSome = action(() => {});`,
+        },
+        {
+            code: `const doSome = action(() => {}, "do");`,
+        }
+    ],
+    invalid: [
+        {
+            code: `
+            import { action } from '@reatom/framework'
+            const doSome = action();
+            `,
+            errors: [{ message: 'action "doSome" should has a name inside action() call' }],
+            output: `
+            import { action } from '@reatom/framework'
+            const doSome = action("doSome");
+            `
+        },
+        {
+            code: `
+            import { action } from '@reatom/framework'
+            const doSome = action("do");
+            `,
+            errors: [{ message: `action "doSome" should be named as it's variable name, rename it to "doSome"` }],
+            output: `
+            import { action } from '@reatom/framework'
+            const doSome = action("doSome");
+            `
+        },
+        {
+            code: `
+            import { action } from '@reatom/framework'
+            const doSome = action(() => {});
+            `,
+            errors: [{ message: `action "doSome" should has a name inside action() call` }],
+            output: `
+            import { action } from '@reatom/framework'
+            const doSome = action(() => {}, "doSome");
+            `
+        },
+        {
+            code: `
+            import { action } from '@reatom/framework'
+            const doSome = action(() => {}, "do");
+            `,
+            errors: [{ message: `action "doSome" should be named as it's variable name, rename it to "doSome"` }],
+            output: `
+            import { action } from '@reatom/framework'
+            const doSome = action(() => {}, "doSome");
+            `
+        }
+    ]
+});
+
+
+tester.run('reatom/reatom-prefux-rule', reatomPrefixRule, {
+    valid: [
+        {
+            code: `
+            import { reatomAsync } from '@reatom/framework'
+            const fetchUser = reatomAsync(() => {}, "fetchUser");
+            `
+        },
+        {
+            code: `
+                import { reatomAsync } from '@reatom/framework'
+                const fetchUser = reatomAsync(() => {}, {
+                    name: "fetchUser",
+                    onRequest: () => {},
+                  });
+              `
+        },
+        {
+            code: `
+            import { reatomAsync } from '@reatom/framework'
+            const fetchUser = reatomAsync(() => {}, { name: "fetchUser" });
+            `
+        },
+        {
+            code: `const fetchUser = reatomAsync(() => {});`,
```

Comment:
We should handle **all** `reatom*` methods, it doesn't metter what import it from.

Reply:
Well, again, no democracy((
Ok, mb it's better